src/lib/shopping-event-conversions.js

```diff
@@ -28,18 +28,17 @@ function convertShoppingEventToFptiInput(
   eventType : EventType
 ) : FptiInput {
 
-  if (!event.user) {
-    event.user = config.user;
-  }
-
   const storedUserIds = getStoredUserIds();
 
+  const eventDataPayload = { ...event };
```

Comment:
why we need to do this, Please, share examples

Reply:
I'm doing this for the next part where I remove `user_id` from the payload so it is not repeated in `sinfo`.

https://github.com/paypal/paypal-muse-components/pull/124/files#diff-eacd74ffe70966f9d40b06ba2c6584ad610922a4cde7772f73d360f39f75469cR34